daemon/event.go

```diff
@@ -92,6 +94,34 @@ func (d *daemon) CreateEvent(req *pb.CreateEventRequest, resp pb.Daemon_CreateEv
 		return fmt.Errorf("user credentials could not found on context %v", err)
 	}
 
+	// checking through eventPool is not good enough since booked events are not added to eventpool
+	isEventExist, err := d.dbClient.IsEventExists(ctx, &pbc.GetEventByTagReq{
+		EventTag: req.Tag,
+		// it will take INVERT condition which means that query from
+		//Running, Suspended and Booked events
+		Status: Closed,
+	})
+	if err != nil {
+		noEventExist := fmt.Errorf("event does not exist or something is wrong: %v", err)
+		resp.Send(&pb.LabStatus{ErrorMessage: noEventExist.Error()})
+		return noEventExist
+	}
+	if isEventExist.IsExist {
+		resp.Send(&pb.LabStatus{ErrorMessage: NotAvailableTag})
+		return fmt.Errorf(NotAvailableTag)
```

Comment:
Why the format?

Reply:
This is not done in this PR, there are general inconsistencies of error handling on the platform itself. Can be handled with a separate issue.

Reply:
Okay, if you want it to be handled in another issue i think it should be created

Reply:
Arranging error handling guideline would be preferred way to go in my opinion. Like we should use `fmt.Errorf` in these particular situation otherwise `errors.New` and etc. Your proposal an issue would be nice to see @Lanestolen :)